src/bgw/job_stat.c

```diff
@@ -333,6 +337,8 @@ bgw_job_stat_tuple_mark_end(TupleInfo *ti, void *const data)
 	fd->last_run_success = result_ctx->result == JOB_SUCCESS ? true : false;
 	fd->total_crashes--;
 	fd->consecutive_crashes = 0;
+	// last_crash_reported = false;
+	fd->flags = ts_clear_flags_32(fd->flags, LAST_CRASH_REPORTED);
```

Comment:
Comment just duplicates code, so not needed. It is clear as it is.
```suggestion
	fd->flags = ts_clear_flags_32(fd->flags, LAST_CRASH_REPORTED);
```